src/sentry/interfaces/csp.py

```diff
@@ -29,11 +29,51 @@
     "'none'", "'self'", "'unsafe-inline'", "'unsafe-eval'",
 ))
 
+DIRECTIVES = frozenset((
+    'base-uri', 'child-src', 'connect-src', 'default-src',
+    'font-src', 'form-action', 'frame-ancestors',
+    'img-src', 'manifest-src', 'media-src', 'object-src',
+    'plugin-types', 'referrer', 'reflected-xss',
+    'script-src', 'style-src', 'upgrade-insecure-requests',
+
+    # Deprecated directives
+    # > Note: This directive is deprecated. Use child-src instead.
+    # > https://developer.mozilla.org/en-US/docs/Web/Security/CSP/CSP_policy_directives#frame-src
+    # 'frame-src',
+
+    # I don't really know what this even is.
+    # 'sandbox',
+))
+
 ALL_SCHEMES = (
     'data:', 'mediastream:', 'blob:', 'filesystem:',
     'http:', 'https:', 'file:',
 )
 
+SELF = "'self'"
+
+DIRECTIVE_TO_MESSAGES = {
```

Comment:
@benvinegar We'll want to try and fill in these human friendly messages. It's a bit unclear to me how to message the ones I haven't filled in. :fearful: